### DISCUSSION TITLE
[Discussion] Removing the community ledger and putting pardons on a fixed cooldown

### DISCUSSION THREAD

**Discussion by zfbx** [Crowns-of-Power/CoP#8] [Ideas]
So the community ledger is something that seems to be a love/hate sort of system in game and I'd like to suggest it be removed and replaced with a different system. From my understanding it's not the most reliable and doesn't work sometimes?

**New suggestion:**
When you commit a dishonorable murder you'll get your normal murderer state (red name) but rather than going to the community to beg for pardon instead I'm suggesting you will keep your murder state for 3-7 days (undecided atm) from your last dishonorable murder and once that time is up you'll be pardoned automatically by the system.

**Pros:**
- This would take some strain off the servers constantly checking pardons and remove some extra fluff from the database which could have some performance benefits.
- This would prevent people from using alts to pardon themselves or prevent others from being pardoned
- This would make a fair balance between punishment for people giving them fixed sentencing 

**Cons:**
- You wont be able to prevent people from being pardoned.

_Because this is a rather big change I'd like to get the thoughts of the community either by leaving a constructive comment below or voting with a thumbs up or down to this post. Also if you have a suggestion for how long the murder state should last before being pardoned please comment with your idea_

**Comment by zfbx** [comment]
@Omega4200 That is what I was saying in the suggestion. so if you kill dishonorably with a few hours to go it completely pushes your time back
 > you will keep your murder state for 3-7 days (undecided atm) from **your last dishonorable murder**

Now I've asked Darrell to post his suggestion but he's being a lazy shit so I'll add it here:
> its a dice system so u kill someone and on Friday or what ever day we have it set (to reset) it will roll a number based on how many ppl u kill so if u killed one person then maybe u have a 90% to roll above 90 or at 90 to be pardon and then the chance goes down for every death you cause
Which isn't a bad idea, little more complex but I will say.. RNG in CoP is SHT lmao

**Comment by allanon-pilgrim** [comment]
I would lean towards the fixed term penalty of 7 days from last dishonourable kill ( if less than 7 days, people that only play weekends can murder someone and then by the following weekend, already be pardoned and ready to kill again with no notable sanctions). Could possibly add a point store Item "Reduced sentence" that if purchased reduces it to 5 days (one item per pardon). The item stays in effect until pardon is activated.

**Comment by ShonkyDonkey** [comment]
I would like this to be changed also, voting system is of no use and very biased and cheat-able. 
Instead i would start with 3 hours CD and than double it up after every kill of white person.
IMO red players should not have safety anywhere but maybe DC, and should be rewarded with higher drop rates from mobs/bosses.
Also maybe consider to disable looting in non pvp bags when person is red but only enable loot if there is room in pvp bag.

**Comment by Omega4200** [comment]
36 hour timer and each time u kill your timer restarts..... /tableflip

**Comment by TehNo0b** [comment]
I think right now theirs no point in even caring if your red/orange/white.  I think pardons are pointless, the whole ledger is.  I like the simple timer idea of turning white again rather than RNG.  But I'd also like to see Guards beat the shit out of murderers keeping our street safe at night from all these criminals.  I want to see perks for innocent/outlaw/murderer statuses each having their own purpose and benefit.    If we can make the whole murderer system be less about who has more friends and/or alts we can really step up what this whole system can mean.  Let's please not stop with simple removal of ledger, and putting timers on pardons.  This is a great opportunity to expand this whole system.

**Reply by TurtleScribe** [reply]
Definitely should add more benefits and negatives to being a murderer/innocent.
